mods_display.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |gem|
   gem.executables = gem.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'stanford-mods', '~> 3.3', '>=3.3.8' # require stanford-mods 3.3.8 for publisher field
+  gem.add_dependency 'stanford-mods', '~> 3.3', '>=3.3.9' # require stanford-mods 3.3.8 for edition field
```

Comment:
Your comment needs to be updated. (I'm not sure this sort of a comment is really necessary, as pinning to `>=` implies a requirement; what that requirement is doesn't really matter.)